frontend/src/main/components/Commons/EditCommonsForm.js

```diff
@@ -0,0 +1,123 @@
+import React, {useState} from 'react'
+import { Button, Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { useNavigate } from 'react-router-dom'
+
+//export default function EditCommonsForm(props) {
+export default function EditCommonsForm(props, { initialCommons }) {
+  const { onSubmit } = props;
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm(
+    { defaultValues: initialCommons || {}, }
+  );
+
+  const navigate = useNavigate();
+
+  return (
+
+   <Form onSubmit={handleSubmit(onSubmit)}>
+
+      <Form.Group className="mb-3">
+        <Form.Label htmlFor="name">Commons Name</Form.Label>
+        <Form.Control
+          id="name"
+          type="text"
+          isInvalid={!!errors.name}
+          {...register("name", { required: "Commons name is required" })}
+        />
+        <Form.Control.Feedback type="invalid">
+          {errors.name?.message}
+        </Form.Control.Feedback>
+      </Form.Group>
+ { /*
+  //     <Form.Group className="mb-3">
+  //       <Form.Label htmlFor="startingBalance">Starting Balance</Form.Label>
+  //       <Form.Control
+  //         id="startingBalance"
+  //         type="number"
+  //         step="0.01"
+  //         isInvalid={!!errors.startingBalance}
+  //         {...register("startingBalance", {
```

Comment:
What is the reason for all of this commented out code?

Reply:
deleted commented out lines